ai-machine-learning/thoth-thamos/advise.md

```diff
@@ -0,0 +1,49 @@
+Thamos CLI runs based on the `.thoth.yaml` config, which we have setup already in the example repository here. 
+
+You can browse the config file in the editor or via terminal using -
+
+``cat .thoth.yaml``{{execute}} 
+
+Now we are ready to do ask Thoth for Advise -
+
+``thamos advise``{{execute}}
```

Comment:
When I tried to run this in my Katacoda env, I got this as my output:
![Screenshot from 2020-04-07 15-07-56](https://user-images.githubusercontent.com/7343099/78709507-c23d6680-78e1-11ea-965c-c79920d88c63.png)

Reply:
Sorry about that, we are facing some issues with our prod env, should be fixed soon.

Reply:
Hey @saisankargochhayat any updates on this?

I had a server timeout error when I tried to run this scenario
![Screenshot from 2020-04-14 11-01-15](https://user-images.githubusercontent.com/22333506/79240510-e3510a80-7e3f-11ea-9606-84967501cefc.png)

Reply:
@4n4nd  can you test that again, it been fixed.

Reply:
works now :tada: 

![Screenshot from 2020-04-14 14-28-36](https://user-images.githubusercontent.com/7343099/79260465-510b2f80-7e5c-11ea-822a-84e3d781443f.png)